pebble/src/test/java/com/mitchellbosecke/pebble/CoreFiltersTest.java

```diff
@@ -135,6 +135,27 @@ void testDate() throws ParseException, PebbleException, IOException {
     assertEquals("07/01/20122012-July-12012/July/1", writer.toString());
   }
 
+  @Test
+  void testDateWithAlterDate() throws ParseException, PebbleException, IOException {
```

Comment:
Ideally, we would split these tests, not sure it would be worth it though. They were already there, but without the new `alterTime` argument.